jsonb-generator/src/test/java/io/avaje/jsonb/generator/models/valid/ignore/IgnoredClass.java

```diff
@@ -0,0 +1,18 @@
+package io.avaje.jsonb.generator.models.valid.ignore;
+
+import java.math.BigDecimal;
+
+import io.avaje.jsonb.Json;
+import io.avaje.jsonb.Json.Ignore;
+
+@Json
+public class IgnoredClass {
```

Comment:
This isn't actually ignoring at the type though ... so whats up here?

Reply:
What happens when you code too late at night